app/controllers/api/v1/split_details_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Api::V1::SplitDetailsController < AuthenticatedApiController
+  def show
+    @split_detail = SplitDetail.new(split: Split.find_by!(name: params[:id]))
```

Comment:
So we have two-ish places that are doing this now. Should we go ahead and make a reader on the `Split` model for `detail`? then this would become `split.find_by!(...).detail` and the controller for editing split details would be able to use the same pattern. Does that makes sense to you?